extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -319,7 +319,7 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
     const TAG = this.getName_();
     const ampAdResourceId = user().assertString(
-        getAmpAdResourceId(this.element, this.win.top),
+        getAmpAdResourceId(this.element, this.win.__AMP_TOP),
```

Comment:
nit: you can use `getTopWindow()` method from service.js

Reply:
Ok, will do. Thanks!